tests/testUtil.py

```diff
@@ -17,7 +17,7 @@
 def createUniqueWARC():
     lines = []
     warcInFilename = 'frogTest.warc'
-    warcInPath = os.path.join(os.path.dirname(__file__) + '/samples/warcs/' +
+    warcInPath = os.path.join('samples/warcs/' +
```

Comment:
Are these relative paths stable? I feel like they will be resolved based on the context from where the tests are run and not the location of the file itself. This is just a guess, I have not verified it. A safer approach would be something like `os.path.dirname(__file__) + '/../samples/warcs/`.

Reply:
@ibnesayeed For pytest, yes, they should be stable. The context should be maintained for each test at the root of the package.

Reply:
Allow me to test something.

Reply:
Test to your heart's content. This PR has already been merged.

Reply:
As I guessed, it breaks miserably when tested from anywhere other than the root of the repository. While our automated test is set to run from that context, there is no reason why these references should not be made more robust. We might want to create a variable to hold the absolute path of the samples directory and use it in different places as necessary to construct more specific paths:

```py
samples_dir = os.path.join(os.path.dirname(__file__) + '/../samples/'
```

Reply:
Opened a separate ticket (#553) for this.